### ISSUE TITLE
Upper Case and large inputs

### ISSUE THREAD

**Issue by cam-parra** [kyle-west/truthfulness#8]
page doesn’t work with uppercases and not sure if this input `(p ∨ (p ∨ (((p ∨ q) ∨ q) ∨ (p ∨ q))))` evaluates correctly

**Comment by kyle-west** [comment]
@cam-parra is this in `Expanded` or `Inline` view?

**Comment by kyle-west** [comment]
@cam-parra I fixed the uppercase variable issue. Also, I think whatever was making the `(p ∨ (p ∨ (((p ∨ q) ∨ q) ∨ (p ∨ q))))` results incorrect was fixed with my Issue #4 resolution. Here is the output now:

**Expanded View**
```
| p | q | p∨q | (p∨q)∨q | ((p∨q)∨q)∨(p∨q) | p∨(((p∨q)∨q)∨(p∨q)) | p∨(p∨(((p∨q)∨q)∨(p∨q))) |
|---+---+-----+---------+-----------------+---------------------+-------------------------|
| F | F |  F  |    F    |        F        |          F          |            F            |
| F | T |  T  |    T    |        T        |          T          |            T            |
| T | F |  T  |    T    |        T        |          T          |            T            |
| T | T |  T  |    T    |        T        |          T          |            T            |

```

**Inline View**
```
| ( | p | ∨ | ( | p | ∨ | ( | ( | ( | p | ∨ | q | ) | ∨ | q | ) | ∨ | ( | p | ∨ | q | ) | ) | ) | ) |
|---+---+---+---+---+---+---+---+---+---+---+---+---+---+---+---+---+---+---+---+---+---+---+---+---|
|   | F | F |   | F | F |   |   |   | F | F | F |   | F | F |   | F |   | F | F | F |   |   |   |   |
|   | F | T |   | F | T |   |   |   | F | T | T |   | T | T |   | T |   | F | T | T |   |   |   |   |
|   | T | T |   | T | T |   |   |   | T | T | F |   | T | F |   | T |   | T | T | F |   |   |   |   |
|   | T | T |   | T | T |   |   |   | T | T | T |   | T | T |   | T |   | T | T | T |   |   |   |   |

```

Both seem to be evaluating correctly, so I will close this issue. If you feel this is still incorrect, please let me know. Thanks again for finding this. Because of it, I was able to locate and fix another two unknown bugs.
